src/task.c

```diff
@@ -819,6 +816,21 @@ task_config_valid(const task_system_config *cfg)
    return STATUS_OK;
 }
 
+platform_status
```

Comment:
There is one remaining call to `task_system_config_init()`, which now can return an `rc`, that still needs to be checked in the caller. See the call in `functional/test.h` in `test_config_init()`. Minor cleanup pending ...